src/Apcu/bootstrap.php

```diff
@@ -26,3 +26,13 @@ function apcu_inc($key, $step = 1, &$success = false) { return apc_inc($key, $st
     function apcu_sma_info($limited = false) { return apc_sma_info($limited); }
     function apcu_store($key, $var = null, $ttl = 0) { return apc_store($key, $var, $ttl); }
 }
+
+if (!class_exists('APCUIterator', false)) {
```

Comment:
also, `APCIterator` is available from `PECL apc >= 3.1.1`, so it doesn't exist on 3.0 for example